Doc/library/sqlite3.rst

```diff
@@ -73,27 +73,26 @@ This example uses the iterator form::
 
 .. _sqlite3-placeholders:
 
-Usually your SQL operations will need to use values from Python variables.  You
-shouldn't assemble your query using Python's string operations because doing so
-is insecure; it makes your program vulnerable to an SQL injection attack
-(see the `xkcd webcomic <https://xkcd.com/327/>`_ for a humorous example of
-what can go wrong)::
+SQL operations will usually need to use values from Python variables. It is poor
+practice to assemble queries using Python's string operations, because they are
+vulnerable to SQL injection attacks (see the `xkcd webcomic
+<https://xkcd.com/327/>`_ for a humorous example of what can go wrong)::
 
    # Never do this -- insecure!
    symbol = 'RHAT'
    cur.execute("SELECT * FROM stocks WHERE symbol = '%s'" % symbol)
 
-Instead, use the DB-API's parameter substitution. Put a placeholder wherever
-you want to use a value, and then provide a tuple of values as the second
+Instead, use the DB-API's parameter substitution by putting placeholders in the
+query string. Provide the actual values as a :class:`tuple` as the second
 argument to the cursor's :meth:`~Cursor.execute` method. An SQL statement may
```

Comment:
```suggestion
Instead, use the DB-API's parameter substitution. To insert a variable into
a query string, use a placeholder in the string, and substitute the actual
values into the query by providing them as a :class:`tuple` of values to
the second argument of the cursor's :meth:`~Cursor.execute` method. An
SQL statement may
```

Reply:
Let me think a little bit about this.

Reply:
Applied. I find it easier to read now. Thanks!

Reply:
@erlend-aasland My pleasure 🙂 (If you fancy returning the favour on my PR for the `GenericAlias` docs, I'd be very grateful... but no worries if not!!)

Reply:
I'll see if I can have a look tomorrow!